ehsm_kms_service/ehsm_kms_params.js

```diff
@@ -106,7 +106,28 @@ const ehsm_kms_params = {
       maxLength: _6kb_length,
```

Comment:
so do you want to change the above length check? they are also not correct as i told you previously.

Reply:
or you can do them in another patch for the code refine

Reply:
It will be fix in another patch.